<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_node</class>
 <widget class="QDialog" name="new_node">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start node</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Name of the node</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_ip">
     <property name="text">
      <string>&amp;Local interface to run node (eg. localhost, IP, hostname)</string>
     </property>
     <property name="buddy">
      <cstring>ip</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ip">
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_port">
     <property name="text">
      <string>Local port, &amp;where node will be started</string>
     </property>
     <property name="buddy">
      <cstring>port</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="port">
     <property name="minimum">
      <number>1025</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chck_init">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Do you want to connect to other node ? </string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_ip_next">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>IP (or hostname) of node you want to connect to</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ip_next">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_port_next">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Port of node you want to connect to</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="port_next">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1025</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_keys">
     <property name="text">
      <string>Load file with keys</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>new_node</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>new_node</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chck_init</sender>
   <signal>toggled(bool)</signal>
   <receiver>ip_next</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chck_init</sender>
   <signal>toggled(bool)</signal>
   <receiver>port_next</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chck_init</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_ip_next</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chck_init</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_port_next</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
